README.md

```diff
@@ -8,14 +8,14 @@
 
 Showcase your app to new users or explain functionality of new features.
 
-It uses [react-floater](https://github.com/gilbarbara/react-floater) for positioning and styling.  
```

Comment:
The double space is required at EOL

Reply:
Hah - I am so used to my editor doing that I didn't question it.   Darn Markup mode configed wrong!